File: UI/CinemaApplication/src/app/entities/vision-movie-update-vm.ts
import { MovieHouse } from './movie-house';
import { Session } from './session';
import { Movie } from './movie';

export class VisionMovieUpdateVM {
    id: number;
    movieId: number;
    sessionId : number;
    movieHouseId: number;
    displayDate: Date;
    price: number;
}

File: CinemaApplication.Entity/ViewModels/VisionMovie/VisionMovieUpdateVM.cs
using CinemaApplication.Entity.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace CinemaApplication.Entity.ViewModels
{
   public class VisionMovieUpdateVM : UpdateVMBase
    {
        public long MovieId { get; set; }
        public long SessionId { get; set; }
        public long MovieHouseId { get; set; }
        public string DisplayDate { get; set; }
        public double Price { get; set; }
    }
}
